Update kn-plugins.md (#4684)
knative/docs: docs/install/client/kn-plugins.md

The `kn` CLI supports the use of plugins. Plugins enable you to extend the functionality of your `kn` installation by adding custom commands and other shared commands that are not part of the core distribution of `kn`.

!!! warning

>     The plugins must be named with a `kn-` prefix in order to be detected by `kn`. `kn-func` will be detected but `func` won't be detected.

Reviewer: ```suggestion
    The plugins must be named with the prefix `kn-` to be detected by `kn`. For example, `kn-func` will be detected but `func` won't be detected.
```
